pkg/synopsysctl/cmd_deploy.go

```diff
@@ -91,6 +91,13 @@ var deployCmd = &cobra.Command{
 
 		log.Infof("deploying the Synopsys-Operator in namespace %s...", deployNamespace)
 
+		// if image has no tag then add "latest"
+		imageHasTag := len(strings.Split(synopsysOperatorImage, ":")) == 2
+		if !imageHasTag {
+			synopsysOperatorImage = fmt.Sprintf("%s:latest", synopsysOperatorImage)
```

Comment:
`synopsysOperatorImage` is already declared up as `var synopsysOperatorImage = "docker.io/blackducksoftware/synopsys-operator:2019.4.0-RC"`.  You can just use that, right?  Instead of latest?

Reply:
Actually, it's worse than that.  Will will result in docker.io/blackducksoftware/synopsys-operator:2019.4.0-RC:latest, which isn't valid. :)

Reply:
This value gets overwritten if the user provides a value. Thus it now just requires the user to enter a tag.